src/components/ui/Card/Card.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+
+export interface CardProps {
+  image: JSX.Element
+  header: string | JSX.Element
+  body: string | JSX.Element
+  styles?: string
+}
+
+export default function Card({
+  header,
+  image,
+  body,
+  styles = 'bg-white drop-shadow rounded-lg'
```

Comment:
Can you please use  the following Daisy-UI classes (so that we can support additional themes in the future)? Thanks!
- bg-white --> `bg-base-100` 
- rounded-lg --> `rounded-box`